src/libopensc/card-epass2003.c

```diff
@@ -1766,6 +1804,12 @@ epass2003_init(struct sc_card *card)
 		exdata->bFipsCertification = 0x00;
 	}
 
+	if (memcmp(&data[3],"\x81\x02\x1D\xD4",4) == 0) {
+		exdata->bFips2 = 0x01;
+	}
+	else {
+		exdata->bFips2 = 0x00;
+	}
```

Comment:
I thin this change needs some comment what does it do and why.

Reply:
This comes from comparing multiple epass22003 tokens, color, name listed on the token, ATRs and the first data read from the card which has parameters.  Various people contributed data from the epass2003 cards as discussed in #3034 and #2843. 

Here is an excel spreadsheet of the differences between the cards. The cards with the "81" tag with "1DD4" appear to be the  "Fips 2" cards, where the vendor changed the applet, and RSA 2048 sign does not work, but EC does.
  
[230303-Feitian-tokens.xlsx](https://github.com/user-attachments/files/16311236/230303-Feitian-tokens.xlsx)